### PR TITLE
add path, boundary to iam roles; add path to iam policies

### PR DESCRIPTION
**Pull request by leslie-corbalt**
This PR follows the docs [here](https://cloud.cms.gov/creating-identity-access-management-policies) that explain that each iam_role and iam_policy must have the `path` attribute set to `/delegatedadmin/developer/` . This setting allows users with cloudtamer roles to have sufficient iam permissions.

I set the boundary value to the permissions boundary arn that is associated with my cloudtamer role, rather than the one documented. Per Rabee Zuberi, `the boundary policy will attach automatically.` so it's value in terraform doesn't matter.

With this, I can now modify the password-rotation scheduled task terraform, as necessary.

Tested:
Verified I could change a container environment variables (`KEY`), and add new environment variables, which destroy/create task definition. I could not do this before.

### CODE REVIEW

File: password-rotation/main.tf [hunk 1]
```diff
@@ -1,5 +1,7 @@
 locals {
   awslogs_group = "/aws/ecs/${var.app_name}-${var.environment}-${var.task_name}"
+  iam_path      = "/delegatedadmin/developer/"
```
Comment (ben-harvey) [inline password-rotation/main.tf:3]: Maybe add a comment with a link to the CCG docs here for future reference?

**Review by ben-harvey — COMMENTED** [review]
As we discussed in sync, if the `iam_boundary` that you specify truly doesn't matter since it is applied automatically, it might be clearer to just omit it from the terraform code.  But if you do so you should probably test that everything still works when it's omitted

**Comment by leslie-corbalt** [comment]
> .
Replying to your comment re permissions boundary: I decided to set the value to exactly what the document says. I include a comment that references the docs. If, in fact, the boundary value is optional, then setting this value per the docs should work. I will update this comment after I test.

**Review by ben-harvey — APPROVED** [review]
LGTM!

**Comment by leslie-corbalt** [comment]
#15
